src/framework/uicomponents/view/abstractmenumodel.cpp

```diff
@@ -257,6 +257,9 @@ void AbstractMenuModel::setItem(int index, MenuItem* item)
 MenuItem& AbstractMenuModel::item(MenuItemList& items, const QString& itemId)
 {
     for (MenuItem* menuItem : items) {
+        if (!menuItem) {
+            continue;
+        }
```

Comment:
Maybe this should be an `IF_ASSERT_FAILED` now, because IIRC I fixed the root cause of this problem so that the check should succeed nowadays.